src/repos/UserRepo.ts

```diff
@@ -1,13 +1,32 @@
 import { getConnectionManager, Repository } from 'typeorm';
 import { UserUpdateFields } from '../common/types';
 import User from '../entities/User';
+import LogUtils from '../utils/LogUtils';
 
 const db = (): Repository<User> =>
   getConnectionManager()
     .get()
     .getRepository(User);
 
-const initalizeUser = async (
+/**
+* Creates a dummy user and saves it to the db (Testing purposes)
+* @function
+* @param {string} id - Google id to create user with
+* @return {User} New dummy user
+*/
+const createDummyUser = async (id: string): Promise<User> => {
+  try {
+    const dummyUser = User.dummy(id);
+    let user = await getUserByNetID(dummyUser.netID);
```

Comment:
`const`?

Reply:
Oops yea, I caught this in the next PR so this gets changed there.